src/components/TextField/TextField.tsx

```diff
@@ -22,14 +23,20 @@ export const TextField: React.FC<Props> = ({
   label = name,
   placeholder = `Enter ${label}`,
   required = false,
-  onChange = () => {},
+  onChange = () => { },
+  validate = () => true,
 }) => {
   // generage a unique id once on component load
   const [id] = useState(() => `${name}-${getRandomDigits()}`);
 
   // To show errors only if the field was touched (onBlur)
   const [touched, setTouched] = useState(false);
   const hasError = touched && required && !value;
```

Comment:
```suggestion
  const hasError = touched && required && !value.trim();
```
it will fix such problem as in screen. I can fill some required inputs with only spaces and no see any errror, but still can't to add new movie.

![chrome-capture-2024-1-13 (2)](https://github.com/mate-academy/react_movies-list-add-form/assets/33029864/60ad0770-ea58-417b-80c7-7022cfc06716)